### PR TITLE
Interlock fix - remove haproxy and custom template files

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

**Comment by paigehargrave** [comment]
@trapier @ehazlett PTAL - thanks!

**Comment by ehazlett** [comment]
LGTM

**Comment by paigehargrave** [comment]
@ehazlett Thank you! @bermudezmt - Can you review and merge? Thanks!

File: _data/toc.yaml [hunk 1]
```diff
@@ -1323,10 +1323,6 @@ manuals:
           section:
           - title: Configure your deployment
             path: /ee/ucp/interlock/config/
-          - title: Using a custom extension template
```
Comment (bermudezmt) [inline _data/toc.yaml]: @paigehargrave @ehazlett Is the decision to add information on custom templates to a KB article?

**Comment by paigehargrave** [comment]
@trapier - should the custom template info go into base?

**Comment by trapier** [comment]
LGTM.  We'll discuss where to post the custom template procedure as a follow-up.

**Comment by bermudezmt** [comment]
Sounds good everyone. I'm merging this PR for now. Thank you, @paigehargrave .

**Pull request merged**